python/ray/_private/ray_constants.py

```diff
@@ -222,6 +222,14 @@ def env_set_by_user(key):
     "%(asctime)s\t%(levelname)s ({component}) %(filename)s:%(lineno)s -- %(message)s"
 )
 
+# A dict_keys object containing the standard attributes of a LogRecord. This is used to
+# help us determine which attributes constitute user-provided context.
+LOGRECORD_STANDARD_ATTRS = logging.makeLogRecord({"message": "test"}).__dict__.keys()
```

Comment:
If we use `makeLogRecord({})` to create a LogRecord, the LogRecord will not have the `message` attribute.